src/server/scripts/Northrend/Ulduar/Ulduar/boss_freya.cpp

```diff
@@ -594,8 +594,8 @@ class boss_freya : public CreatureScript
                 const uint32 summonSpell[2][4] =
                 {
                               /* 0Elder, 1Elder, 2Elder, 3Elder */
-                    /* 10N */    {62950, 62953, 62955, 62957},
-                    /* 25N */    {62952, 62954, 62956, 62958}
+                    /* 10N */    {62957, 62955, 62953, 62950},
+                    /* 25N */    {62958, 62956, 62954, 62952}
```

Comment:
No, I was looking contents of chest by chest (using .cast)
I thought about going retracing the contents by DB, Aokromes who asked to test if their order was not incorrect. When testing the order, I noticed that she was inverted.

About Sniff unfortunately no one is dispois to share.